pkg/aws/ec2.go

```diff
@@ -119,6 +133,29 @@ func AssertEC2VolumeEncrypted(t *testing.T, ctx context.Context, client EC2Clien
 	assert.True(t, deviceEncrypted, "Volume with device ID '%s' for instance '%s' was not encrypted.", input.DeviceID, input.InstanceID)
 }
 
+// AssertVolumeType asserts the right volume type & associated throughput
+func AssertEC2VolumeType(t *testing.T, ctx context.Context, client EC2Client, input AssertEC2VolumeTypeInput) {
```

Comment:
**Suggestion:** Rather than having a struct with "default" values, have you considered taking the functional options approach, with one options func for each possible assertion (i.e. one for type, one for IOPS, and one for throughput)? It would lead to a cleaner and more extensible interface.

Reply:
Addressed this as well.